### PR TITLE
[CASSANDRA-17023] Removed duplicate toCQLString in ReadCommand

### PR DESCRIPTION
**Pull request by subkanthi**
[CASSANDRA-17023] Removed duplicate toCQLString in ReadCommand

### CODE REVIEW

File: src/java/org/apache/cassandra/db/ReadCommand.java [hunk 1]
```diff
@@ -781,35 +781,6 @@ protected LongPredicate getPurgeEvaluator()
...
-     * debugging purpose which is what this is for.
-     */
-    public String toCQLString()
-    {
-        StringBuilder sb = new StringBuilder().append("SELECT ")
-                                              .append(columnFilter().toCQLString())
-                                              .append(" FROM ")
-                                              .append(ColumnIdentifier.maybeQuote(metadata().keyspace))
-                                              .append('.')
-                                              .append(ColumnIdentifier.maybeQuote(metadata().name));
```
Comment (maedhroz) [inline src/java/org/apache/cassandra/db/ReadCommand.java:800]: nit: `import org.apache.cassandra.cql3.ColumnIdentifier;` is now unused above as a result of removing this.

**Review by maedhroz — APPROVED** [review]
LGTM (w/ the exception of one small nit)

File: src/java/org/apache/cassandra/db/ReadCommand.java (hunk 1 above)
Comment (subkanthi) [reply src/java/org/apache/cassandra/db/ReadCommand.java:800]: Thanks, Removed.

**Comment by maedhroz** [comment]
Apache CI: https://ci-cassandra.apache.org/view/patches/job/Cassandra-devbranch/1227/

**Comment by maedhroz** [comment]
@subkanthi Once you have a second +1, I would squash this down to 1 commit if it's not too much trouble. Thanks!

**Comment by maedhroz** [comment]
@subkanthi Just a note, to squash, you can `git rebase -i <the SHA of the commit before yours>` and then `git push --force-with-lease`

**Comment by subkanthi** [comment]
> @subkanthi Just a note, to squash, you can `git rebase -i <the SHA of the commit before yours>` and then `git push --force-with-lease`

Thanks will try, definitely struggling with the commands.

**Comment by subkanthi** [comment]
> > @subkanthi Just a note, to squash, you can `git rebase -i <the SHA of the commit before yours>` and then `git push --force-with-lease`
> 
> Thanks will try, definitely struggling with the commands.

Thanks @maedhroz , the commands worked.

**Comment by maedhroz** [comment]
Committed in https://github.com/apache/cassandra/commit/60e0da1bd0f8271f7bbba300695d67b9ad0a497f

**Pull request closed without merging**